packages/terra-worklist-data-grid/src/subcomponents/ColumnResizeHandle.jsx

```diff
@@ -4,7 +4,9 @@ import React, {
 import PropTypes from 'prop-types';
 import { injectIntl } from 'react-intl';
 import classNames from 'classnames/bind';
+
```

Comment:
There really is not a reason to change this file.

Reply:
Reverted: [69036d9](https://github.com/cerner/terra-framework/pull/1710/commits/69036d994769ea38abc772f887a266894c8f9517)